src/cockpit/389-console/src/lib/ldap_editor/lib/utils.jsx

```diff
@@ -258,7 +260,10 @@ export function getSearchEntries (params, resultCallback) {
       if (err.exit_status === 4) {
         console.log('Size limit hit'); // Use the partial data.
         searchResult = data;
-        // TODO: Check other relevant error codes ( 32, 53 ...)
+        params.addNotification(
+            "info",
+            `Size limit of ${params.sizeLimit} was exceeded.  The child entries of "${params.searchBase}" have been truncated.`
```

Comment:
I think it makes sense to mention in the help message that "Full list of entries can be accessed through search or table view". Otherwise, I think, we can hear a lot of questions like - "Can we configure the limit?", "Is it a planned feature to expand the tree view", etc.
So a more detailed info message will help here, IMHO.

Reply:
Actually it can only be done through the search page.  Table and tree both use the hardcoded size limit

Reply:
I'll update the other message when the search is done through the the tree/table.

Reply:
changes made, merging....